ios/Exponent/Kernel/DevSupport/EXSession.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXSession : NSObject
+
++ (instancetype)sharedInstance;
+
+- (NSDictionary * _Nullable)getSession;
```

Comment:
```suggestion
- (NSDictionary * _Nullable)session;
```

Obj-C / Swift convention is to use noun phrases for non-mutating getters (whereas in JS / Java, methods => verb phrases)